.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -469,6 +469,7 @@ def write(x=''):
 config = OrderedDict()
 config['homework_library_enabled'] = False
 config['homework_library_location'] = os.path.join(SUBMITTY_INSTALL_DIR, 'library')
+config['homework_library_allowed'] = False
```

Comment:
Looks like the same thing as homework_library_enabled. Maybe change to something like homework_library_manager_enabled. Or it could be the required level so they could put 2 to allow instructors and admins, or 1 for admins, or 0 for nobody.